ui/pages/confirmations/components/confirm/info/hooks/useDecodedTransactionData.ts

```diff
@@ -23,6 +26,7 @@ export function useDecodedTransactionData(
 
   return useAsyncResult(async () => {
     if (
+      !isDecodeEnabled ||
```

Comment:
For the sake of immediately resolving the privacy regression, is there anything more needed than just the above two lines?

Reply:
Afraid so, the redesigned confirmations are reliant on the advanced decoding meaning they would lose core functionality if it were disabled.

Reply:
Afraid so, fundamental data such as token transfer recipient and approve spending cap.